### ISSUE TITLE
Training : Streaming Platforms for Training Events?

### ISSUE THREAD

**Issue by geomantic** [osmus/teachosm#10]
Has the OSM organizations looked into streaming platforms such as Twitch.tv?

**Comment by geomantic** [comment]
Using Webex through GWU, which is very full-featured, but requires a bit of pre-event d/l & config to get it working for end users. 
OSMUS uses Zoom but with the plan we have, we're limited to 40-min session.

**Comment by iandees** [comment]
> we're limited to 40-min session

Nope, we have a license for unlimited time on Zoom. If you want to use the license we have you just have to pick it up from whoever has it currently.

**Comment by geomantic** [comment]
Zoom & Webex (via GWU) have worked fine. Perhaps talk to Martijn to get his take on BigBlueButton and Twitch.
